pkg/plugins/resources/helm/utils.go

```diff
@@ -147,12 +150,18 @@ func (c *Chart) MetadataUpdate(chartPath string, dryRun bool) error {
 			md.Version = v.IncMinor().String()
 		case PATCHVERSION:
 			md.Version = v.IncPatch().String()
+		case NOINCREMENT:
+			// Reset Version to its initial value
+			md.Version = oldVersion
+			break forLoop
 		default:
-			logrus.Errorf("Wrong increment rule %q.", inc)
+			logrus.Warningf("Wrong increment rule %q, ignoring", inc)
 		}
 	}
 
-	logrus.Infof("\tChart Version updated from %q to %q\n", oldVersion, md.Version)
+	if oldVersion != md.Version {
+		logrus.Infof("\tChart Version updated from %q to %q\n", oldVersion, md.Version)
```

Comment:
Should a log be also included if md.AppVersion has changed as well?

Reply:
It's already done on line 127

Reply:
WDYT of putting the block l125-129 below this one?